src/python/pants/testutil/skip_utils.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2022 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
The python skips could probably be moved here. I wanted to keep the diff minimal though.